docs/operator-manual/argocd-cm.yaml

```diff
@@ -428,4 +431,4 @@ data:
   webhook.maxPayloadSizeMB: 1024
 
   # application.sync.impersonation.enabled indicates whether the application sync can be decoupled from control plane service account using impersonation.
-  application.sync.impersonation.enabled: "false"
+  application.sync.impersonation.enabled: "false"
```

Comment:
I doubt you want to remove the end of line from the end of the file

Reply:
Oops! My bad. Must have removed it while cleaning up the code. Will fix it.

Reply:
fixed it.